axolotl-web/src/components/AddContactModal.vue

```diff
@@ -9,7 +9,7 @@
           </button>
         </div>
         <div class="modal-body">
-          <p><strong>Only add numbers that you are sure are registered Signal accounts.</strong></p>
+          <span v-translate><strong>Only add numbers that you are sure are registered Signal accounts.</strong></span>
```

Comment:
the v-translate have to be put into the <strong> because we don't want to expose to much html tags to the translators

Reply:
Done.